sonic_platform_base/sfp_base.py

```diff
@@ -0,0 +1,204 @@
+        keys                       |Value Format   |Information	
+        ---------------------------|---------------|----------------------------
+        RX LOS                     |BOOLEAN        |rx lost-of-signal status,
+                                   |               |True if has rx los, False if not.
+        TX FAULT                   |BOOLEAN        |tx fault status,
+                                   |               |True if has tx fault, False if not.
+        Reset status               |BOOLEAN        |reset status,
+                                   |               |True if SFP in reset, False if not.
+        LP mode                    |BOOLEAN        |low power mode status,
+                                   |               |True in lp mode, False if not.
+        tx disable                 |BOOLEAN        |tx disable status,
+                                   |               |True tx disabled, False if not.
+        tx disabled channel        |HEX            |disabled TX channles in hex,
+                                   |               |bits 0 to 3 represent channel 0
+                                   |               |to channel 3.
+        ========================================================================
+        """
+        return NotImplementedError
+                
+    def get_reset_status(self):
+        """
+        Retrieves the reset status of SFP
+
+        Returns:
+            A Boolean, True if reset enabled, False if disabled
+        """
+        return NotImplementedError
+
+    def get_rx_los(self):
+        """
+        Retrieves the rx los (lost-of-signal) status of SFP
+
+        Returns:
+            A Boolean, True if SFP has rx los, False if not.
+            Note : rx los status is lached until a call to get_rx_los or a reset.
+        """
+        return NotImplementedError
+
+    def get_tx_fault(self):
+        """
+        Retrieves the tx fault status of SFP
+
+        Returns:
+            A Boolean, True if SFP has tx-fault, False if not
+            Note : tx fault status is lached until a call to get_tx_fault or a reset.
+        """
+        return NotImplementedError
+
+    def get_tx_disable(self):
+        """
+        Retrieves the tx-disable status of this SFP
+
+        Returns:
+            A Boolean, True if tx-disable is enabled, False if disabled
+        """
+        return NotImplementedError
+
+    def get_tx_disable_channel(self):
+        """
+        Retrieves the tx disabled channels in this SFP
+
+        Returns:
+            A hex of 4 bits (bit 0 to bit 3 as channel 0 to channel 3) to represent
+            tx channels which have been disabled in this SFP.
+            As an example, a returned value of 0x5 indicates that channel 0 
+            and channel 2 have been disabled.
+        """
+        return NotImplementedError
+
+    def get_lpmode(self):
+        """
+        Retrieves the lpmode status of this SFP
```

Comment:
Move the parenthetical full description `(low power mode)` from the return value comment on line 125 below to this line, as this comes first.

Reply:
Moved full descriptions.